pyproject.toml

```diff
@@ -53,6 +53,62 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+alignment = [
+    "dtw-python",
+    "numba",
+]
+annotation = [
+    "attrs",
+    "hmmlearn",
+    "numba",
+    "pyod",
+]
+classification = [
+    "esig",
+    "mrsqm",
+    "numba",
+    "tensorflow",
+    "tsfresh",
```

Comment:
and tslearn too?

Reply:
I never used classification, but VS Code search couldn't find that against `python_dependencies` tag. Can you please point out the relevant line/module/sub-package?

Reply:
hm, indeed, there are no classifiers from `tslearn` currently, upon checking.

Which is odd, since `tslearn` is a significant time series classification package, and I thought we had at least a few interfaces there.

Reply:
Closing this as resolved, as it's not being used.